lib/widgets/profile.dart

```diff
@@ -0,0 +1,272 @@
+    }
+  }
+}
+
+class ProfilePage extends StatelessWidget {
+  final int userId;
+
+  const ProfilePage({super.key, required this.userId});
+
+  static Route<void> buildRoute({required BuildContext context, required int userId}) {
+    return MaterialAccountWidgetRoute(context: context,
+      page: ProfilePage(userId: userId));
+  }
+
+  Widget buildErrorPage() {
+    return Scaffold(
+      appBar: AppBar(title: const Text("Error")),
+      body: const SingleChildScrollView(
+        child: Padding(
+          padding: EdgeInsets.all(16),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Icon(Icons.error),
+              Text("Could not show user profile."),
+            ]))));
+  }
+
+  void _createDmNavigation (context) {
+    final store = PerAccountStoreWidget.of(context);
+    final allRecipientIds = [store.account.userId, userId];
+    allRecipientIds.sort();
+    final narrow = DmNarrow(
+      selfUserId: store.account.userId,
+      allRecipientIds: allRecipientIds);
+    Navigator.push(context,
+      MessageListPage.buildRoute(context: context, narrow: narrow));
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final store = PerAccountStoreWidget.of(context);
+    final user = store.users[userId];
+    if (user != null) {
+      final avatar = Avatar(userId: userId, size: 200, borderRadius: 200 / 8);
+      final Widget profileDataTable;
+      if (user.profileData != null) {
+        profileDataTable = ProfileDataTable(user:user);
+      } else {
+        profileDataTable = const SizedBox.shrink();
+      }
+      final Widget localTimeWidget;
+      if (!user.isBot) {
+        final userLocalNow = getNowInTimezone(user.timezone);
+        if (userLocalNow != null) {
+          localTimeWidget = Text("${DateFormat.jm().format(userLocalNow)} Local time",
+            style: const TextStyle(fontWeight: FontWeight.normal));
+        } else {
+          localTimeWidget = const SizedBox.shrink();
+        }
+      } else {
+        localTimeWidget = const SizedBox.shrink();
+      }
+      return Scaffold(
+        appBar: AppBar(title: Text(user.fullName)),
+        body: SingleChildScrollView(
+          child: Padding(
+            padding: const EdgeInsets.all(16),
+            child: DefaultTextStyle.merge(
+              textAlign: TextAlign.center,
+              style: const TextStyle(fontSize: 20, height: 1.5),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.stretch,
+                children: [
+                  Center(child: avatar),
+                  const SizedBox(height: 16),
+                  Text(user.fullName,
+                    style: const TextStyle(fontWeight: FontWeight.bold)),
+                  (user.isBot ? Text(user.email) : const SizedBox.shrink()),
+                  Text(user.role.label),
+                  // Text("Active about XXX"), // TODO render active status
```

Comment:
This TODO comment can point to the relevant issue:
```suggestion
                  // Text("Active about XXX"), // TODO(#196) render active status
```